setup.py

```diff
@@ -23,9 +22,15 @@
 
 
 # determine CPU support for SSE2 and AVX2
-cpu_info = cpuinfo.get_cpu_info()
-have_sse2 = 'sse2' in cpu_info['flags']
-have_avx2 = 'avx2' in cpu_info['flags']
+try:
+    import cpuinfo
```

Comment:
Unfortunately this actually won't benefit from `setup_requires` at this stage as Setuptools hasn't had the opportunity to run this. To get this to work correctly, we need to add a [`BuildExtCommand` subclass like this one for NumPy]( https://github.com/jakirkham/cookiecutter-cython/blob/de7397f77ba5118c9e5973966f925d2a4af220a7/%7B%7Bcookiecutter.project_name%7D%7D/setup.py#L21-L27 ) and add it to [our `cmdclasses`]( https://github.com/jakirkham/cookiecutter-cython/blob/de7397f77ba5118c9e5973966f925d2a4af220a7/%7B%7Bcookiecutter.project_name%7D%7D/setup.py#L73 ) in [`setup`]( https://github.com/jakirkham/cookiecutter-cython/blob/de7397f77ba5118c9e5973966f925d2a4af220a7/%7B%7Bcookiecutter.project_name%7D%7D/setup.py#L159 ).

Reply:
Doh, of course.